bpf/lib/bpf_cgroup.h

```diff
@@ -7,7 +7,7 @@
 #include "hubble_msg.h"
 #include "bpf_helpers.h"
 #include "environ_conf.h"
-#include "bpf_helpers.h"
+#include "process.h"
```

Comment:
The commit message of the first patch, mentions:
> by its pinned Cgroup ID.

It is not clear what the "pinned Cgroup ID" is. Some of the parts of the commit message seem to also
be out-of-date.

I suggest to use the following commit message for the first patch:

> This patch introduces the cgroup id tracker in the execve_map value.
> The goal is to track the cgroup id of the container that every process belongs to.
>
> As a first step, this patch retrieves the value of the cgroup id whenever a new task is created (fork),
> and inserts it into the execve_map. A subsequent patch will change the exec hook to use this value.

Reply:
Pinned cgroup ID could be the ID of container part of k8s, a systemd service, or a container outside of k8s. So not necessarily a container.

Improve the commit message.